frontend/src/CONFIG.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const EINREICHFRIST_FUENFTE_PK = '1.1.2022';
```

Comment:
Ist das hier benötigt? Soll ja jetzt im backend gespeichert werden

Reply:
Wird in der Trelokarte: https://trello.com/c/8tHgleA1/77-abgabedatum-5-pk-dynamisch-aus-api-beziehen dann umgesetzt. jetzt bleibt das erstmal so